core/src/main/scala/fs2/netty/Network.scala

```diff
@@ -163,8 +164,8 @@ object Network {
 
     (instantiateR("server"), instantiateR("client")) mapN { (server, client) =>
       try {
-        val meth = eventLoopClazz.getDeclaredMethod("setIoRatio", classOf[Int])
-        meth.invoke(server, new Integer(90))    // TODO tweak this a bit more; 100 was worse than 50 and 90 was a dramatic step up from both
+        val meth = eventLoopClazz.getDeclaredMethod("setIoRatio", classOf[Int]) // TODO: this method is set to be depcrecated in the future releases
```

Comment:
According to [Epoll source](https://github.com/netty/netty/blob/9cac18687d15be113cf869426f4bd1aca804b6b0/transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java#L140) this doesn't even do anything. For [KQueue](https://github.com/netty/netty/blob/9cac18687d15be113cf869426f4bd1aca804b6b0/transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueEventLoopGroup.java#L140) and [NIO](https://github.com/netty/netty/blob/9cac18687d15be113cf869426f4bd1aca804b6b0/transport/src/main/java/io/netty/channel/nio/NioEventLoopGroup.java#L126) ELG this has a meaningful impact. As to what's replacing Epoll's method...🤷‍♂️ ha. So it's not really a universal interface for ELG's, but Netty makes it seem like it is.

One takeaway is that this creates a discrepancy between tuning on non-linux OS'es vs. other OS'es.

Reply:
Oh this is really interesting!

I'm not *that* fussed about the tuning being a bit different on different OSes, though if it starts getting to the point where people might actually take that into account and build their applications and profiling to those kinds of parameters, then it might be an issue. At some point, differences between platforms are very much unavoidable. Like if you're profiling `kqueue` and using it to tune a server running against `epoll` or (even worse) `io_uring`, you're going to have a bad time with or without our meddling.

Reply:
👍